nixos/modules/services/misc/forgejo.nix

```
@@ -322,6 +322,12 @@ in
                 description = "Disable external SSH feature.";
               };
 
+              START_SSH_SERVER = mkOption {
+                type = types.bool;
+                default = false;
+                description = "When enabled, use the built-in SSH server.";
```

Nit, non-blocking, but how about
```suggestion
                description = ''
                  When enabled, use Forgejo's built-in SSH server instead of the
                  hosts' existing SSH server ({option}`services.openssh.enable`).

                  ::: {.note}
                  This likely requires manually changing the value of either
                  {option}`services.openssh.ports` or
                  {option}`services.forgejo.settings.server.SSH_LISTEN_PORT` so they don't collide.
                  :::
                '';
```

fyi you can build the manual locally using `nix-build nixos/release.nix -A manual.x86_64-linux`

---

Updated this description as well